CRM/Contact/DAO/SavedSearch.php

```diff
@@ -250,6 +292,95 @@ public static function &fields() {
           'serialize' => self::SERIALIZE_JSON,
           'add' => '5.24',
         ],
+        'created_id' => [
+          'name' => 'created_id',
+          'type' => CRM_Utils_Type::T_INT,
+          'title' => ts('Created By Contact ID'),
+          'description' => ts('FK to contact table.'),
+          'where' => 'civicrm_saved_search.created_id',
+          'table_name' => 'civicrm_saved_search',
+          'entity' => 'SavedSearch',
+          'bao' => 'CRM_Contact_BAO_SavedSearch',
+          'localizable' => 0,
+          'FKClassName' => 'CRM_Contact_DAO_Contact',
+          'html' => [
+            'label' => ts("Created By"),
+          ],
+          'add' => '5.36',
+        ],
+        'modified_id' => [
+          'name' => 'modified_id',
+          'type' => CRM_Utils_Type::T_INT,
+          'title' => ts('Modified By Contact ID'),
+          'description' => ts('FK to contact table.'),
+          'where' => 'civicrm_saved_search.modified_id',
+          'table_name' => 'civicrm_saved_search',
+          'entity' => 'SavedSearch',
+          'bao' => 'CRM_Contact_BAO_SavedSearch',
+          'localizable' => 0,
+          'FKClassName' => 'CRM_Contact_DAO_Contact',
+          'html' => [
+            'label' => ts("Modified By"),
+          ],
+          'add' => '5.36',
+        ],
+        'expires_date' => [
+          'name' => 'expires_date',
+          'type' => CRM_Utils_Type::T_TIMESTAMP,
+          'title' => ts('Search Expiry Date'),
+          'description' => ts('Optional date after which the search is not needed'),
+          'required' => FALSE,
+          'where' => 'civicrm_saved_search.expires_date',
+          'table_name' => 'civicrm_saved_search',
+          'entity' => 'SavedSearch',
+          'bao' => 'CRM_Contact_BAO_SavedSearch',
+          'localizable' => 0,
+          'add' => '5.36',
+        ],
+        'created_date' => [
+          'name' => 'created_date',
+          'type' => CRM_Utils_Type::T_TIMESTAMP,
+          'title' => ts('Created Date'),
+          'description' => ts('When the search was created.'),
+          'required' => TRUE,
+          'where' => 'civicrm_saved_search.created_date',
+          'default' => 'CURRENT_TIMESTAMP',
+          'table_name' => 'civicrm_saved_search',
+          'entity' => 'SavedSearch',
+          'bao' => 'CRM_Contact_BAO_SavedSearch',
+          'localizable' => 0,
+          'add' => '5.36',
+        ],
+        'modified_date' => [
+          'name' => 'modified_date',
+          'type' => CRM_Utils_Type::T_TIMESTAMP,
+          'title' => ts('Modified Date'),
+          'description' => ts('When the search was lase modified.'),
```

Comment:
Typo: "lase"

Reply:
This one is actually supposed to be modified with lasers.

Reply:
I dunno if we should add lasers as a hard-dependency. We want to support older systems that don't have built-in CD burners. IMO Civi should work on any box fast enough to run Win95.

Reply:
OK - you had your fun - all lasers updated now